packages/cactus-plugin-ledger-connector-ethereum/src/main/typescript/plugin-ledger-connector-ethereum.ts

```diff
@@ -304,6 +304,9 @@ export class PluginLedgerConnectorEthereum
     app: Express,
     wsApi: SocketIoServer,
   ): Promise<IWebServiceEndpoint[]> {
+    // Add custom replacer to handle bigint responses correctly
+    app.set("json replacer", this.stringifyBigIntReplacer);
+
```

Comment:
@eduv09 This is one step in the right direction but now that I realize what the whole picture was: This is actually adding another issue, meaning that this change will make it work in the test environment but when it's used in production (e.g. through the API server) then it will conflict with the API server's own big int replacer implementation. So, the solution IMO is to move this json replacer registration code to the test cases themselves which then guarantees that it only happens where it's needed and doesn't cause trouble in the other scenario (API server usage)

Reply:
Ok. In that case maybe we can close this PR, and leave this issue to be solved in the future the way you suggested in the previous comment (long term solution).
In the meantime I will add the replacer in my tests that use the connector

Reply:
@eduv09 OK, that works for me too, thanks and sorry for the back and forth!